src/main/java/com/jduk/api/controller/ApiController.java

```diff
@@ -34,18 +33,34 @@ public ResponseEntity<List<Album>> getAllAlbums() {
     // GET album by id
     @Operation(summary = "Retrieve album information from database record by associated ID.")
     @GetMapping("/{id}")
-    public ResponseEntity<Album> getAlbumByID(@PathVariable @Parameter(name = "ID", description = "Unique ID associated with a particular album of choice.") Long id){
+    public ResponseEntity<Album> getAlbumByID(@PathVariable("id") @Parameter(name = "ID", description = "Unique ID associated with a particular album of choice.") Long id){
         return new ResponseEntity<>(apiService.getAlbumByID(id), HttpStatus.OK);
     }
 
     // DELETE album by id
     @Operation(summary = "Delete record of album from database by associated ID.")
     @GetMapping("/{id}")
-    public ResponseEntity<Album> deleteAlbumByID(@PathVariable @Parameter(name = "ID", description = "Unique ID associated with a particular album of choice.") Long id){
+    public ResponseEntity<Album> deleteAlbumByID(@PathVariable("id") @Parameter(name = "ID", description = "Unique ID associated with a particular album of choice.") Long id){
         return new ResponseEntity<>(apiService.deleteAlbumByID(id), HttpStatus.OK);
     }
 
+    // POST album
```

Comment:
Excellent.  No need to over-complicate the endpoint logic at this moment in time.